specs/order_spec.rb

```diff
@@ -142,20 +148,58 @@
 
     it "Returns accurate information about the last order" do
       # TODO: Your test code here!
+      id = 100
+      products = {
+        "Amaranth" => 83.81,
+        "Smoked Trout" => 70.6,
+        "Cheddar" => 5.63
+      }
+      customer_id = 20
+      fulfillment_status = :pending
+
+      order = Order.all.last
+
+      # Check that all data was loaded as expected
+      expect(order.id).must_equal id
+      expect(order.products).must_equal products
+      expect(order.customer).must_be_kind_of Customer
+      expect(order.customer.id).must_equal customer_id
+      expect(order.fulfillment_status).must_equal fulfillment_status
     end
   end
 
   describe "Order.find" do
     it "Can find the first order from the CSV" do
       # TODO: Your test code here!
+      first = Order.find(1)
+
+      expect(first).must_be_kind_of Order
+      expect(first.id).must_equal 1
     end
 
     it "Can find the last order from the CSV" do
       # TODO: Your test code here!
+      first = Order.find(100)
+
+      expect(first).must_be_kind_of Order
+      expect(first.id).must_equal 100
     end
 
-    it "Returns nil for an order that doesn't exist" do
+  it "Returns nil for an order that doesn't exist" do
       # TODO: Your test code here!
+      non_existant_order = Order.find(300)
+
+      expect(non_existant_order).must_be_nil
+    end
+  end
+
+  it "Returns a list of Order instances where the value of the customer's ID matches the passed parameter" do
+    orders = []
+    orders = Order.find_by_customer(10)
```

Comment:
What happens if you look up orders for a customer who has no orders, or for a customer that doesn't exist?